sys/txs-compiler/test/data/success/MultiActions.txs

```diff
@@ -0,0 +1,22 @@
+{-
+TorXakis - Model Based Testing
+Copyright (c) 2015-2017 TNO and Radboud University
+See LICENSE at root directory of this repository.
+-}
+
+PROCDEF p [Input :: Int] () ::=
+       ISTEP
+   >-> Input ? x
+   >-> Input ? x | ISTEP
```

Comment:
Should we have illegal examples?
I prefer not.
Solution is to add another input channel and use it here, e.g. Input ? x | AnotherInput ? y.

Reply:
It was my idea that whether this is legal or not is under discussion. If this example should be rejected by the compiler then we should make an issue out of it, because right now it accepts it.

I cannot change it in the way you suggest, because the error was triggered by having the predefined actions combined with channel offers. If you would run the example you suggest on `develop`, as far as I remember, it will pass.

Also note that these are not examples meant for the users of TorXakis, but rather test cases for the compiler.